models/articles.model.js

```diff
@@ -1,7 +1,38 @@
 const db = require("../db/connection")
 
 
-function fetchArticleID (article_id) {  
+
+exports.fetchArticles = () => {
+    return db.query(
+`SELECT   
+    articles.article_id, 
+    articles.title, 
+    articles.topic, 
+    articles.author, 
+    articles.created_at, 
+    articles.votes, 
+    articles.article_img_url,
+    COUNT(comments.article_id) AS comment_count
+FROM 
+    articles 
+JOIN 
+    comments ON articles.article_id = comments.article_id
+GROUP BY
+    articles.article_id
+ORDER BY 
+    articles.created_at DESC;` 
+    ).then((articles) => {
+        return articles
```

Comment:
Would better to return the ready data `articles.rows`
or destructure rows on line 24 - `{rows}` instead of `articles` then return `rows`